src/components/SVGViewer/interfaces.ts

```diff
@@ -53,6 +53,14 @@ interface SvgViewerBasicProps {
    * Override default colors with custom classNames
    */
   customClassNames?: CustomClassNames;
+  /**
+   * Maximum zoom for the svg viewer
+   */
+  maxZoom?: number;
```

Comment:
```suggestion
  maxZoom?: number = 30;
```
move default values here

Reply:
I couldn't seem to add the default values here (without getting errors), so I added default props to the SVGViewer instead.

Reply:
ah yeah, my bad. this is because it's interface
your solution is good